lib/frontman/commands/serve.rb

```diff
@@ -84,7 +85,35 @@ def serve
 end
 
 class FrontManServer < Sinatra::Base
-  set :port, 4568
+  port = default_port = Frontman::Config.get(:port, fallback: 4568)
+  num_retries = Frontman::Config.get(:port_retries, fallback: 1)
+
+  port_retry_strategy = Frontman::Config.get(:port_retry_strategy, ->(port_num){
```

Comment:
We need to declare `port_in_use` outside of the scope of our block, or it won't be available for the error check later on.

```suggestion
  port_retry_strategy = Frontman::Config.get(:port_retry_strategy, fallback: ->(port_num){
      port_in_use = false
```

Reply:
Fixed